admin/client/App/screens/List/index.js

```diff
@@ -104,6 +105,14 @@ const ListView = React.createClass({
 				case 'sort':
 					this.props.dispatch(setActiveSort(query[key]));
 					break;
+				case 'filters':
+					try {
+						const filters = JSON.parse(query[key]);
+						setActiveFilters(filters);
+					} catch (e) {
+						console.warn('Invalid filter provided');
```

Comment:
is there no way to surface this in the ui? maybe using a flash message or a popup or something?